.semaphore/semaphore.yml

```diff
@@ -22,10 +23,16 @@ blocks:
             - sudo dpkg -i /tmp/libpng12.deb
             - sudo touch /etc/apt/sources.list.d/google-chrome-source.list
             - install-package --update-new google-chrome-stable
+            - gem install semaphore_test_boosters
             - cache restore
             - yarn install
             - bundle install --deployment --path vendor/bundle
             - cache store
             - 'bundle exec rake db:setup'
             - 'bundle exec rake db:test:prepare'
-            - bundle exec rspec
+            - rspec_booster --job $SEMAPHORE_JOB_INDEX/$SEMAPHORE_JOB_COUNT
+promotions:
+  - name: Sandbox Deployment
+    pipeline_file: sandbox_deployment.yml
+    auto_promote:
+      when: branch = 'master' AND result = 'passed'
```

Comment:
This `auto_promote` bit is what triggers the Sandbox deployment automatically on successful `master` builds.